.vscode/launch.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Attach to Node Functions",
+      "type": "node",
+      "request": "attach",
+      "port": 9229,
+      "preLaunchTask": "func: host start"
+    }
+  ]
+}
```

Comment:
I've validated that this correctly loads up a debugger (though it did do it on the 2nd API call, not the first)